sql/glam-fenix-dev/glam_etl/org_mozilla_fenix_glam_nightly__clients_histogram_aggregates_v1/query.sql

```diff
@@ -68,6 +72,7 @@ filtered_accumulated AS (
       @submission_date,
       INTERVAL 365 day
     )
+    AND app_version >= (latest_version - 3)
```

Comment:
Doesn't this retain the last 4 versions and not 3 like the config indicates?

Reply:
Yes, I believe you're right.